### PR TITLE
fix TI-RTOS build and time() related issues

### PR DESCRIPTION
**Pull request by tisb-vikram**
These commits contain fixes for incorrect time value and the elfcc32 build issues.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@tisb-vikram__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@tisb-vikram__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: c/doc/run_sample_on_ti_cc3200.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ While not strictly required, we recommend that you install the following tools f
 
 - Install [TI-RTOS SDK for SimpleLink](http://downloads.ti.com/dsps/dsps_public_sw/sdo_sb/targetcontent/tirtos/index.html) 2.14.01.20 or above
 
-- Install [NS package](http://software-dl.ti.com/dsps/dsps_public_sw/sdo_sb/targetcontent/ns/ns_1_10_00_00_eng.zip). Please note, the NS package is a pre-release version. It's content and the download location are subject to change.
+- Install [NS package](http://software-dl.ti.com/dsps/dsps_public_sw/sdo_sb/targetcontent/ns/ns_1_10_00_03.zip). Please note, the NS package is a pre-release version. It's content and the download location are subject to change.
```
Comment (tameraw) [inline c/doc/run_sample_on_ti_cc3200.md]: is the ns_1_10_00_03, still a pre-release version? If not, can you please update the text in that note? Thanks.

**Comment by tameraw** [comment]
@tisb-vikram - This has been merged into develop. Thanks for your contribution.

**Pull request closed without merging**